imagetest/fixtures.go

```diff
@@ -156,3 +159,30 @@ func (t *TestVM) EnableSecureBoot() {
 		}
 	}
 }
+
+// AddTestUser generate ssh key pair and return public key.
+func (t *TestVM) AddTestUser(user string) ([]byte, error) {
+	email := fmt.Sprintf("%s@example.com", user)
```

Comment:
i dont really understand this. you're only using it in the optional 'comment' field of the key. i don't think you need this to be in email format?

Reply:
Yes, I did notice that this is a comment field. It seems that's not necessary to have.